react/react dependency installer/installer/e4e1b2e6-26a1-4b3b-b472-4464814fa69a.json

```diff
@@ -0,0 +1,51 @@
+{
+  "implementations": {
+    "es6-node": {
+      "inputFunctions": {
+        "045a62ad-7340-479a-b544-f10d56f16597": "const { exec } = require( \"child_process\" );\nconst { dependencies } = params;\nif (data.start !== false ){\n  const command = \"cd \" + data.projectPath + \" && \" + \"npm install \" + params.dependencies.join(' ');\n  outputs.stdout( command, tag );\n  if (state.running){\n    outputs.bounced(data, tag);\n  } else {\n    state.running = true;\n    exec(command, (error, stdout, stderr) => {\n      if (error) {\n        outputs.error(`${error}`, tag);\n      } else {\n        outputs.stdout(stdout || \"\", tag);\n        outputs.stderr(stderr || \"\", tag);\n        outputs.error(null, tag);\n      }\n      state.running = false;\n    });\n  }\n}"
```

Comment:
On the callback error branch, we're sending a string instead of the documented `{"error": string}` which is the correct type. Should be something like: `{"error": String(error)}`, or `{"error": "Dependencies failed to install", "details": String(error)}`.